### PR TITLE
Fix failing tests running on python mode

### PR DESCRIPTION
**Pull request by gefarion**

### CODE REVIEW

File: sharedbuffers/mapped_struct.py [hunk 1]
```diff
@@ -422,7 +422,6 @@ def unpack_from(cls, buf, offs, idmap = None):
 # @cython.ccall
 # @cython.returns(cython.bint)
 @cython.locals(_cmp = int)
-@cython.optimize.use_switch(True)
```
Comment (tzulberti-jampp) [inline sharedbuffers/mapped_struct.py:425]: Why did you remove this?

Comment (gefarion) [reply sharedbuffers/mapped_struct.py:425]: It is true by default.

**Review by tzulberti-jampp — APPROVED** [review]

**Review by klaussfreire — APPROVED** [review]
